src/components/ArticleTopics.jsx

```diff
@@ -0,0 +1,27 @@
+import { getAllTopics } from "../utils/api";
+import { useEffect, useState } from "react";
+import { Typography } from "@mui/material";
+import { Link } from "react-router-dom";
+
+export default function ArticleTopics() {
+  const [fetchedTopics, setFetchedTopics] = useState();
+
+  useEffect(() => {
+    getAllTopics().then((data) => {
+      setFetchedTopics(data);
+    });
+  }, []);
+  return (
+    <>
```

Comment:
you can also get rid of this empty tag if Typography is wrapping the whole component anyway